app/api/currentuser/create_group_leave_request.robustness.feature

```diff
@@ -18,8 +18,8 @@ Feature: User creates a request to leave a group - robustness
       | 15              | 21             | null                        |
     And the groups ancestors are computed
     And the database has the following table 'group_pending_requests':
-      | group_id | member_id | type          | at                  |
-      | 14       | 21        | leave_request | 2019-05-30 11:00:00 |
+      | group_id | member_id | type          | at                      |
+      | 14       | 21        | leave_request | 2019-05-30 11:00:00.001 |
```

Comment:
I have the impression that a single case (in robustness?) would be enough to demonstrate milliseconds... otherwise that pollutes the test. (larger column make it more difficult to review on GH for instance)

Reply:
In robustness it's just to check the table stays unchanged, but the DB returns it with milliseconds, so it should be set with milliseconds to be equal